README.md

```diff
@@ -10,6 +10,17 @@ to deploy the [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort) as
 > 
 > <img alt="Eclipse Incubation" src="https://projects.eclipse.org/modules/custom/eclipsefdn/eclipsefdn_projects/images/project_state/incubating.png" width="10%">
 
+## Community
+
+To communicate with the developers, you can:
+* Join the [Matrix chat](https://matrix.to/#/#apoapsis:matrix.eclipse.org).
+* Start a GitHub [discussion](https://github.com/eclipse-apoapsis/ort-server/discussions).
+* Join the [mailing list](https://accounts.eclipse.org/mailing-list/apoapsis-dev).
+
```

Comment:
Should the #ort-server channel in ORT's Slack workspace be added as well for completeness? Or are you planning to deprecated it in favor of the Matrix chat?

Reply:
Yes, I have announced it now in the channel, I didn't find the time earlier.